zebedee-reader/src/main/java/com/github/onsdigital/zebedee/util/URIUtils.java

```diff
@@ -79,11 +83,129 @@ public static String getLastSegment(String uri) {
         return segments[segments.length - 1];
     }
 
+    public static String removeAfterKeyword(String path, String keyword) {
+        if (isEmpty(path)) {
+            return null;
```

Comment:
For all those new functions, you should review very carefully when you are returning null, and decide if returning an empty string would be a better choice. Returning `null` means that now you need to start checking for `null` values in every place where these functions are being used, which is not happening at the moment, and that's going to cause issues.